exercises/concept/vec-macro/.docs/instructions.md

```diff
@@ -0,0 +1,16 @@
+You are going to initialize empty buffers and list the first five numbers of the Fibonacci sequence.
+
+## 1. Create a buffer of `count` zeroes.
+
+Create a function that creates a buffer of `count` zeroes.
+```rust
+let my_buffer = create_buffer(5);
```

Comment:
The example should help the student understand what the expected return value looks like. Is there perhaps a way to show the contents of the vector? See [this F# example](https://github.com/exercism/fsharp/blob/main/exercises/concept/bird-watcher/.docs/instructions.md#1-check-what-the-counts-were-last-week).